### DISCUSSION TITLE
Fork with only 'Write/Code from prompt' using up to date OpenAI API

### DISCUSSION THREAD

**Discussion by idan1633** [MichelNivard/gptstudio#178] [Show and tell]
If you, like me, want to use the Write/Code from Prompt functionality but couldn't find it, I have rolled back to a previous commit with this function and updated it to use the latest version of OpenAIs API. 

https://github.com/idan1633/gptstudio

**Comment by JamesHWade** [comment]
I'm happy to add (or add back / edit) a plugin if it's useful. You may be able to use the "Chat in Source" plugin to do what you describe.
